framework/DataObjects/XDataSet.py

```diff
@@ -1756,7 +1783,7 @@ def _getPathEndings(self):
       # first get rows from collector
       fromColl = self._collector[:][np.where(self._collector[:,self._orderedVars.index('RAVEN_isEnding')])]
       # then turn them into realization-like
-      fromColl = list( dict(zip(self._orderedVars,c)) for c in fromColl if self._orderedVars[c] in self.getVars())
+      fromColl = list( dict(zip(self._orderedVars,c)) for c in fromColl ) #if self._orderedVars[c] in self.getVars())
```

Comment:
We can remove the commented portion, I think.

Reply:
removed